scripts/build.ps1

```diff
@@ -14,40 +14,49 @@ Write-Host "SkipTests=$SkipTests."
 
 if (!(Test-Path "dotnetcli\dotnet.exe")) {
     Write-Host "dotnet.exe not installed, downloading and installing."
-    if ($Version -eq "<default>") {
```

Comment:
`dotnet --list-runtimes` does not exist for 2.0. Do we really need 1.0 and 2.0 installed at all?

Reply:
We probably don't need 1.0 but I am not 100% certain we can remove 2.0. Try to remove them and see if the build fails or if there is any issues with the tests. As far as I can tell, it should be fine to remove them both.

Reply:
We largely keep them for testing purposes, so I would prefer to keep them for now. It would still be good to know that we **can** remove them if we wanted to without unexpected failures.